core/src/com/unciv/logic/civilization/CivInfoStats.kt

```diff
@@ -108,6 +108,17 @@ class CivInfoStats(val civInfo: CivilizationInfo) {
         statMap["Transportation upkeep"] = Stats().apply { gold = -getTransportationUpkeep().toFloat() }
         statMap["Unit upkeep"] = Stats().apply { gold = -getUnitMaintenance().toFloat() }
 
+        if (civInfo.religionManager.religion != null) {
+            for (unique in civInfo.religionManager.religion!!.getFounderBeliefs().flatMap { it.uniqueObjects }) {
+                if (unique.placeholderText == "[] for each global city following this religion") {
+                    statMap.add("Religion", unique.stats.apply {
+                        this.timesInPlace(civInfo.religionManager.numberOfCitiesFollowingThisReligion().toFloat())
+                    })
```

Comment:
`timesInPlace`? side effects intended or guaranteed to not exist because unique is a deep copy already? OK, I'm only looking at the text diff not in Studio, might be easier to check there... But if side effect intended and that flatMap's receiver is a sequence, then wouldn't it go back into the ruleset because the iterator created for uniqueObjects is a view? And... No, my head starts hurting...